app/views/user_journey/view_certificate.html.erb

```diff
@@ -1,8 +1,28 @@
+<% if @certificate.component.signing_certificates.length == 2 %>
+    <div class="govuk-error-summary <%= primary_signing_certificate(@certificate) ? "govuk-error-summary--grey" : "govuk-error-summary--orange" %>" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="error-summary">
+      <h2 class="govuk-error-summary__title" id="error-summary-title"><%= primary_signing_certificate(@certificate) ? t('user_journey.adding_certificate_to_config') : t('user_journey.wait_for_an_email') %></h2>
+      <div class="govuk-error-summary__body">
+        <% if primary_signing_certificate(@certificate) %>
+          <p><%= t 'user_journey.we_will_email_you' %></p>
+          <p><%= t('user_journey.add_new_signing_key_warning', component: display_component(@certificate), date: 'TODO') %></p>
+        <% elsif secondary_signing_certificate(@certificate) %>
+          <p><%= t 'user_journey.how_long_it_takes' %></p>
+          <p><%= t('user_journey.delete_old_signing_key_and_cert', component: display_component(@certificate)) %></p>
+        <% end %>
+        <p><%= render 'shared/component_certificate_doc_links', certificate: @certificate %></p>
+      </div>
+    </div>
+<% end %>
+
 <h1 class="govuk-heading-l">
   <%= @certificate.component.component_type == COMPONENT_TYPE::MSA ? t('user_journey.certificate.msa_heading') : t('user_journey.certificate.vsp_heading') %>
   <%= t('user_journey.certificate.certificate', type: @certificate.usage ) %>
+  <% if !@certificate.encryption? && @certificate.component.signing_certificates.length == 2 %>
+    <%= primary_signing_certificate(@certificate) ? t('user_journey.primary') : t('user_journey.secondary') %>
+  <% end %>
 </h1>
 
 <%= render 'shared/user_certificate_view', certificate: @certificate, full_details: true %>
-
-<%= link_to @certificate.encryption? ? t('user_journey.certificate.replace') : t('user_journey.certificate.add_new'), before_you_start_path, class: "govuk-button", data: { module: "govuk-button" }, role:"button" %>
+<% if @certificate.encryption? || @certificate.component.signing_certificates.length < 2 %>
+  <%= link_to @certificate.encryption? ? t('user_journey.certificate.replace') : t('user_journey.certificate.add_new'), before_you_start_path, class: "govuk-button", data: { module: "govuk-button" }, role:"button" %>
```

Comment:
feels weird we evaluate `@certificate.encryption?` twice right after each other...

Reply:
We can evaluate to signing? instead which reads nicer and just switch the outcomes around but the if statements are needed. Unless theres a better way you can think of to do it?

Reply:
I would probably split them to multiple if statements, first by type and then by the number of certs... 
in pseudo:
```
if signing do
    if certificate.length == 2 do
        #something with 2 certs
    else
       #something with 1 cert
else
      # something encryption